File: src/Generated20260201.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260201 {
public:
    // Generated class 20260201_180110_788781
    int value = 758;
    string created = "20260201_180110_788781";
    
    int process() {
        int result = value;
        for (int i = 0; i < 40; i++) {
            result += i * 4;
        }
        return result;
    }
};

int main() {
    Generated20260201 obj;
    cout << obj.process() << endl;
    return 0;
}

File: src/Generated20260201.go
package main

import "fmt"

type Generated20260201 struct {
    // Generated struct 20260201_175640_246115
    Value int
    Created string
}

func (c *Generated20260201) Process() int {
    result := c.Value
    for i := 0; i < 21; i++ {
        result += i * 10
    }
    return result
}

func main() {
    obj := Generated20260201{Value: 725, Created: "20260201_175640_246115"}
    fmt.Println(obj.Process())
}